packages/cozy-client/src/models/dacc.js

```diff
@@ -30,7 +30,7 @@ export const checkMeasureParams = measure => {
   if (!parsedDate.toISOString().startsWith(startDate)) {
     throw new Error('Date should be in YYYY-MM-DD format')
   }
-  if (!value || typeof value !== 'number') {
+  if (value == null || typeof value !== 'number') {
```

Comment:
nit: J'ai peut-être pas toutes les cartes mais cela devrait suffire ?

```suggestion
  if (typeof value !== 'number') {
``` 

Reply:
Ah! Obviously, good catch